filter-queries/README.md

```diff
@@ -0,0 +1,396 @@
+# Sample CLI application with Filter and Aggregation query examples 
+
+### Background
+
+This CLI Application showcases filter and aggregate queries for `member-profile` Profile Schema using profiles-sdk. 
+We run these queries on already built Profiles, the below table documents the queries:
+
+| Sequence | Query type           | Query                                                                                                                                                                                                                                                                         | About                                           |
+|----------|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------|
+| 1        | Filter (simple)      | cvs_parquet.filter((col("MAIL_STATUS").isin({"MAIL CONTINUE"})) & (col("CASES") == "50"))                                                                                                                                                                                     | Simple Filter                                   |
```

Comment:
1) Maybe we should remove the reference to `"cvs"`? (Here and any other references in URLs and whatnot)

2) The queries in this table don't match what is in `FilterQueries.java` - looks like a different dataset than member profiles.

3) Just my opinion, but without context these queries just look like unformatted code blocks. I think writing the queries in words could make it a bit clearer.